app/src/main/java/org/mozilla/fenix/tabstray/inactivetabs/InactiveTabs.kt

```diff
@@ -0,0 +1,317 @@
+fun InactiveTabsList(
+    inactiveTabs: List<InactiveTabListItem>,
+    expanded: Boolean,
+    showAutoCloseDialog: Boolean,
+    onHeaderClick: () -> Unit,
+    onDeleteAllClick: () -> Unit,
+    onAutoCloseDismissClick: () -> Unit,
+    onEnableAutoCloseClick: () -> Unit,
+    onTabClick: (InactiveTabListItem) -> Unit,
+    onTabCloseClick: (InactiveTabListItem) -> Unit,
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(horizontal = 16.dp, vertical = 12.dp)
+            .clip(ROUNDED_CORNER_SHAPE)
+            .background(FirefoxTheme.colors.layer2)
+            .border(
+                width = 2.dp,
+                color = FirefoxTheme.colors.borderPrimary,
+                shape = ROUNDED_CORNER_SHAPE,
+            )
+            .padding(vertical = 8.dp)
+    ) {
+        InactiveTabsHeader(
+            expanded = expanded,
+            onClick = onHeaderClick,
+            onDeleteAllClick = onDeleteAllClick,
+        )
+
+        if (expanded) {
+            if (showAutoCloseDialog) {
+                InactiveTabsAutoClosePrompt(
+                    onDismissClick = onAutoCloseDismissClick,
+                    onEnableAutoCloseClick = onEnableAutoCloseClick,
+                )
+            }
+
+            Column {
+                inactiveTabs.forEach { tab ->
+                    InactiveTabsItem(
+                        tabTitleText = tab.tabTitleText,
+                        url = tab.displayUrl,
+                        onClick = { onTabClick(tab) },
+                        onCloseClick = { onTabCloseClick(tab) },
+                    )
+                }
+            }
+        }
+    }
+}
+
+/**
+ * Collapsible header for the Inactive Tabs section
+ *
+ * @param expanded Boolean to indicate whether the section is expanded.
+ * @param onClick Called when the user clicks on the header.
+ * @param onDeleteAllClick Called when the user clicks on the delete all button.
+ */
+@Composable
+fun InactiveTabsHeader(
+    expanded: Boolean,
+    onClick: () -> Unit,
+    onDeleteAllClick: () -> Unit,
+) {
+    ExpandableListHeader(
+        headerText = stringResource(id = R.string.inactive_tabs_title),
+        expanded = expanded,
+        expandActionContentDescription = stringResource(R.string.inactive_tabs_expand),
+        collapseActionContentDescription = stringResource(R.string.inactive_tabs_collapse_group),
+        onClick = onClick,
+    ) {
+        Box(
+            modifier = Modifier
+                .clickable(onClick = onDeleteAllClick)
+                .padding(horizontal = 16.dp, vertical = 8.dp),
+        ) {
+            Icon(
+                painter = painterResource(R.drawable.ic_delete),
+                contentDescription = stringResource(R.string.inactive_tabs_delete_all),
+                modifier = Modifier.size(20.dp),
+                tint = FirefoxTheme.colors.iconPrimary,
+            )
+        }
+    }
+}
+
+/**
+ * Inactive Tabs list item Composable
```

Comment:
```suggestion
 * Inactive Tabs list item.
```
Let's drop `Composable`. I think I have been trying to tell you to move away from the internal semantics for your kDocs and go with something more plain that anyone can understand.